docs/pages/machine-id/faq.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Machine ID FAQ
+description: Frequently asked questions about Teleport Machine ID
+---
+
+## Which Teleport features are currently supported by Machine ID?
+
+Broadly, Machine ID currently supports the following Teleport features:
+ * [Server Access](../server-access/introduction.mdx)
+ * [Database Access](../database-access/introduction.mdx)
+ * [Kubernetes Access](../kubernetes-access/introduction.mdx) (*in Teleport v10.1*)
+ * [Application Access](../application-access/introduction.mdx) (*in Teleport v10.1*)
+   * Note: [AWS Console](../application-access/guides/aws-console.mdx) and API access is currently unsupported.
```

Comment:
If we've covered this under the not supported section, do we need it here as well ?

Reply:
I figured it didn't hurt. It could be considered both an unsupported feature, and a caveat for an otherwise supported feature.

Reply:
Fair enough.

Reply:
- I was about to add this comment as well—would it make sense to indicate a particular aspect of Application Access that Machine ID supports in the "does support" list, rather than Application Access as a whole?

- Does "API access" mean "AWS API access"? Otherwise it could be possible to confuse this with "Teleport API Access" directly below.